packages/talos_agent/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
Do we need .gitignore for individual packages ?

Reply:
I dont think so, not sure where did it come from. Good spotting!